static/js/message_list_view.js

```diff
@@ -1163,7 +1163,7 @@ export class MessageListView {
         header.replaceWith(rendered_recipient_row);
     }
 
-    _rerender_message(message_container, message_content_edited, is_revealed) {
+    _rerender_message({message_container, message_content_edited, is_revealed}) {
```

Comment:
I think we want the `message_container` to stay a positional parameter, since that feels more readable -- effectively it's "rerender this message, with these options"; I made that tweak before merging.